crates/viewer/re_component_ui/src/variant_uis/redap_uri_button.rs

```diff
@@ -52,16 +54,45 @@ pub fn redap_uri_button(
             .iter()
             .any(|source| source.redap_uri().as_ref() == Some(&uri));
 
+    let uri_clone = uri.clone();
     // Show the link left aligned and justified, so the whole cell is clickable.
     let put_justified_left_aligned = |ui: &mut Ui, link| {
         ui.scope_builder(
             UiBuilder::new().max_rect(ui.max_rect()).layout(
-                Layout::left_to_right(Align::Center)
-                    .with_main_justify(true)
+                Layout::right_to_left(Align::Center)
                     .with_cross_justify(true)
                     .with_main_align(Align::Min),
             ),
```

Comment:
I believe you could use `egui::Sides` for this. See e.g. how the table headers (which have a right aligned menu button) is implemented, in `header_cell_ui` (`datafusion_table_widget.rs`)